src/helpers/utility.js

```diff
@@ -83,6 +83,19 @@ export function satoshiToDecimal(number) {
   return bn.dividedBy(conversionBN).toNumber();
 }
 
+/**
+ * Converts String to a big number.
+ * @param number {String} The number string to convert.
+ * @return {BigNumber} The converted big number.
+ */
+export function stringToBN(number) {
```

Comment:
please use `web3.utils.toBN()`. BN seems to be preferable over BigNumber since web3 1.0 adopted it.